content/en/index.mdx

```diff
@@ -112,3 +112,4 @@ Help us grow and make the MakerDAO community even better. Reach out in our chat
 
 </InfoBlock>
 
+<Authors all/>
```

Comment:
I would leave all authors on the contribute page, this should def be a community vote.

Reply:
https://chat.makerdao.com/channel/community-development?msg=FgdGPnSf2ipKdpP9y

There was a vote, and it resolved to be the bottom of the front page, that's why I had placed it there for the PR 😅